modules/caddyhttp/responsewriter.go

```diff
@@ -74,15 +74,26 @@ type HTTPInterfaces interface {
 // ResponseWriter does not implement the required method.
 var ErrNotImplemented = fmt.Errorf("method not implemented")
 
+/*
+struct of size 64 bytes could be of size 56 bytes:
+	ResponseWriterWrapper	*github.com/caddyserver/caddy/v2/modules/caddyhttp.ResponseWriterWrapper,
+	statusCode           	int,
+	buf                  	*bytes.Buffer,
+	shouldBuffer         	github.com/caddyserver/caddy/v2/modules/caddyhttp.ShouldBufferFunc,
+	size                 	int,
+	header               	net/http.Header,
+	wroteHeader          	bool,
+	stream               	bool,
+*/
```

Comment:
I do not think this is necessary to keep in the code. The order of fields is seldom changed after the struct is created, especially on unexported types. We can add it to the commit message though!

Reply:
🤦‍♂

Reply:
Removed